model/src/main/java/org/openremote/model/manager/ManagerAppRealmConfig.java

```diff
@@ -27,4 +27,5 @@ public class ManagerAppRealmConfig {
     protected String favicon = null;
     protected String language = null;
     protected String[] headers = null;
+    protected ManagerAppRealmNotificationConfig notifications = null;
```

Comment:
Java variables are `null` by default and assigning `null`  to them again is redundant and results in unnecessary CPU instructions.

See: http://www.javapractices.com/topic/TopicAction.do?Id=14

```suggestion
    protected ManagerAppRealmNotificationConfig notifications;
```